testcafe/tests/pages/viewer/viewer-model.js

```diff
@@ -62,6 +62,11 @@ export default class Viewer {
     )
     for (let i = startIndex; i < startIndex + numberOfNavigation; i++) {
       await this.navigateToNextFile(i)
+      //navigateToNextFile brings you on the next file, so we are sure there is a Previous Button on this image
+      await t.hover(selectors.btnViewerNavPrevious, {
+        offsetX: 0,
```

Comment:
I don't think we need to specify an offset in that case?

Reply:
I think it's best to add an offset when hovering for screenshots, so we are 100% sure the hover will be on exact same spot test after test.
I agree here we don't really need it, as `btnViewerNavPrevious/Next` is not that big so there is not so many chance to trigger another div on page.

So do we keep the offset as best practice or do we remove it here?

Reply:
I think we can keep it. I prefer to set the position manually to be sure where the cursor will be. We already had some issues when not doing that.